distributed/utils.py

```diff
@@ -205,6 +205,22 @@ def All(*args):
     raise gen.Return(results)
 
 
+def is_loop_running(loop):
```

Comment:
Of course, this is not ideal, but gets us running until a decision is made on https://github.com/tornadoweb/tornado/pull/2085